ui-tests/src/test/java/org/openmrs/reference/page/EditReportPage.java

```diff
@@ -0,0 +1,45 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class EditReportPage extends Page {
+	
+	private static final By PREVIEW_BUTTON = By.cssSelector("#previewButton");
+	private static final By START_DATE = By.cssSelector("#startDate");
+	private static final By END_DATE = By.cssSelector("#endDate");
+	private static final String startDate = "05/07/2008";
+	private static final String endDate = "20/06/2020";
+	private static final By RUN_BUTTON = By.cssSelector("#preview-parameterizable-button");
+	private static final By CLOSE_TAB = By.xpath("/html/body/div[4]/div[1]/a");
```

Comment:
Worth getting a better element reference for CLOSE_TAB, the current one can easily break.

Reply:
The current one is too long, using XPath will make it more clear

Reply:
@kdaud   ,Resolved them.Thanks

Reply:
> By.xpath("/html/body/div[4]/div[1]/a");
> using XPath will make it more clear

That xPath() can easily break with those divs.